aws-py-oidc-provider/__main__.py

```diff
@@ -0,0 +1,70 @@
+import pulumi
+from pulumi_aws import iam
+import requests
+import subprocess
+import OpenSSL
+import json
+
+audience = "" # Provide the name of your Pulumi Organization
+oidc_idp_url = 'https://api.pulumi.com/oidc'
+base_url = 'api.pulumi.com/oidc'
+
+# Obtain the OIDC IdP URL and form the configuration document URL
+print("Forming configuration document URL...")
```

Comment:
You probably don't want `print` statements because they will print even if you run the program a second time and nothing changes.

Reply:
Is there a Pulumi-specific way of doing this to where it doesn't print if there's been no changes? I tried with `pulumi.log.info()` and it still does it. I don't want to remove the statements because I think the indicators are helpful for users.